antora-playbook.yml

```diff
@@ -0,0 +1,25 @@
+site:
+  title: devonfw
+  start_page: devon4j::index.adoc
+  url: https://devonfw.com
+content:
+  sources: 
+  - url: .
+    branches: HEAD
+    start_path: .
+    edit_url: 'https://github.capgemini.com/CG-Europe-ABL/devonfw/blob/{refname}/{path}'
```

Comment:
This line needs to be changed to the correct URL

Reply:
Removed this line, as the playbook is anyway only for local testing. Hence, an edit URL is not require.